### DISCUSSION TITLE
page fault

### DISCUSSION THREAD

**Discussion by Bit0r** [nicolargo/glances#2846] [Q&A]
Can `glances` display `page fault` information? Just like the `MAJFLT` and `MINFLT` columns in `htop`.

**Comment by nicolargo** [comment]
Hi @Bit0r 

Glances uses PsUtil to grab system information, for the moment process minor/major page fault feature is not available. 

An enhancement request is open: https://github.com/giampaolo/psutil/issues/1541

I create an issue in Glances to follow the PSUtil implementation.

Nicolas
